app/dts/bindings/zmk,combo.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Combos container
+
+compatible: "zmk,combo"
```

Comment:
I actually prefer the plural for this, since it identifies the parent node containing many combos.

Thoughts?